### PR TITLE
feat(editing-support): add ultimate-autopair.nvim

### PR DESCRIPTION
**Pull request by windowsrefund**

### CODE REVIEW

File: lua/astrocommunity/editing-support/ultimate-autopair-nvim/init.lua [hunk 1]
```diff
@@ -0,0 +1,44 @@
+return {
+  {
+    "windwp/nvim-autopairs",
+    enabled = false,
+  },
+  {
+    "altermo/ultimate-autopair.nvim",
+    lazy = false,
```
Comment (luxus) [inline lua/astrocommunity/editing-support/ultimate-autopair-nvim/init.lua:8]: why did you turn of lazy?

**Comment by Uzaaft** [comment]
@windowsrefund I think the opts should be empty, unless there's a reason why you're setting changing them.

**Review by mehalter — CHANGES_REQUESTED** [review]
Oh, this README.md file should be in the `utlimate-autopair-nvim` folder also can you make the format follow the same as other plugins? This looks like it's more following the format of the README for packs

**Review by mehalter — APPROVED** [review]
LGTM

**Pull request merged**